docs/contribute/01_environment.qmd

```diff
@@ -108,13 +108,13 @@ for manager, params in managers.items():
 
 ### Support matrix
 
-|      Python Version {{< fa arrow-right >}} |                Python 3.9                 |           Python 3.10           |           Python 3.11           |
-| -----------------------------------------: | :---------------------------------------: | :-----------------------------: | :-----------------------------: |
-| **Operating System** {{< fa arrow-down >}} |                                           |                                 |                                 |
-|                                  **Linux** |       {{< fa check >}} [^supported]       |        {{< fa check >}}         |        {{< fa check >}}         |
-|                         **macOS (x86_64)** |             {{< fa check >}}              |        {{< fa check >}}         |        {{< fa check >}}         |
-|                        **macOS (aarch64)** | {{< fa triangle-exclamation >}}[^unknown] | {{< fa triangle-exclamation >}} | {{< fa triangle-exclamation >}} |
-|                                **Windows** |         {{< fa ban >}}[^unlikely]         |         {{< fa ban >}}          |         {{< fa ban >}}          |
+|      Python Version {{< fa arrow-right >}} |  Python 3.9                  |  Python 3.10              |   Python 3.11             |
+| -----------------------------------------: | :--------------------------: | :-----------------------: | :-----------------------: |
+| **Operating System** {{< fa arrow-down >}} |                              |                           |                           |
+|                                  **Linux** | {{< fa check >}}[^supported] | {{< fa check >}}          | {{< fa check >}}          |
+|                         **macOS (x86_64)** | {{< fa check >}}             | {{< fa check >}}          | {{< fa check >}}          |
+|                    **macOS (arm64/M1/M2)** | {{< fa ban >}}[^m1]          | {{< fa ban >}}[^m1]       | {{< fa ban >}}[^m1]       |
```

Comment:
if I type `arch` on my M1 I get "arm64", not aarch64. That's also what python wheels use for M1s.

Reply:
arm64 and aarch64 refer to the same architecture. Nix refers to _any_ ARM-based CPU architecture as aarch64, regardless of whether the operating system is macOS or Linux.